### PR TITLE
search-jobs: update docs for GA

### PR DESCRIPTION
**Pull request by stefanhengl**
- remove beta callout
- new screenshots
- update section "Disable Search Jobs"
- several other minor fixes

### CODE REVIEW

**Comment by stefanhengl** [comment]
@MaedahBatool Should I merge this into a release branch? Search Jobs will be GA with the next release.

**Review by mmanela — APPROVED** [review]

File: docs/code-search/types/search-jobs.mdx [hunk 1]
```diff
@@ -99,20 +96,20 @@ The following result types are not supported:
...
 - Multiple `rev` filters
 - Queries with `index: filter`
 
-<Callout type="note">The search bar already supports exhaustive searches with the `count:all` operator. However, these searches are subject to lower timeouts, making them harder to use in large codebases.</Callout>
 
-## Disable Search Jobs
+<Callout type="note">
+    Alternatively, the search bar supports the `count:all` operator which increases result limits and timeouts.
+    This works well if the search completes within a few minutes and the number of results is less than the configured display limit.
+    For longer running searches and searches with huge results sets, Search Jobs is the better choice.
```
Comment (stefanhengl) [inline docs/code-search/types/search-jobs.mdx:107]:
```suggestion
    For longer running searches and searches with huge result sets, Search Jobs is the better choice.
```

**Review by MaedahBatool — APPROVED** [review]
TY, @stefanhengl. I am approving it, but let's not merge it until I get a working branch for the new release. To avoid the merge, I am making it a draft PR.

**Comment by MaedahBatool** [comment]
@stefanhengl we have the working branch ready. I am merging this PR. :) Thanks once again for contributing.

**Pull request merged**